### PR TITLE
[Estuary] Adjust plot height in video fullscreen info

### PR DESCRIPTION
**Pull request by CrystalP**
## Description

Expand the plot textbox when the next playlist item label is not needed (items played one at a time, or last item of playlist)

This adds one line for the plot. Adding another to completely fill the space would require more significant layout changes.

Q: I don't know much about skinning. Is it worth reducing the number of visibility conditions, and how? Or is there a way to make the existing textbox's height dynamic, and avoid the extra textbox?

## Motivation and context

The space reserved for the next playlist item title under the plot in the info screen is blank when playing items one by one, or when the playlist reached the last item of the playlist.

It looks like there is unused space, and causes some long plots to scroll even when that shouldn't be needed.
For example:
image

## How has this been tested?

Resolutions 640x480 to 4k. With long plots, with playlist and without, at different positions in the playlist.

Tested with local/network files, no PVR or alternate sources available.

## What is the effect on users?

Better use of space in video fullscreen info for long plots.

## Screenshots (if appropriate):
Long plot, single played item or last item of a playlist > textbox with extra line:
image

Long plot, item in middle of playlist with multiple items > no change:
image

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by CrystalP** [comment]
@HitcherUK @enen92 if you don't mind taking a peek. There is probably a more efficient way.

File: addons/skin.estuary/xml/DialogSeekBar.xml [hunk 1]
```diff
@@ -400,26 +400,34 @@
...
+      <control type="grouplist">
+        <left>420</left>
+        <top>10</top>
+        <right>20</right>
+        <height>300</height>
+        <align>left</align>
+        <orientation>vertical</orientation>
+        <itemgap>10</itemgap>
+        <control type="textbox">
+          <height>295</height>
```
Comment (HitcherUK) [inline addons/skin.estuary/xml/DialogSeekBar.xml:412]: Height needs reducing by 1px otherwise you can see the top of the next line when at 1280x720.

**Comment by CrystalP** [comment]
Tabs done.

I tried creating a variable for the visibility condition but got nowhere. I didn't find a wiki page on variables.

**Review by Hitcher — APPROVED** [review]
Looks good, thanks.

**Review by jjd-uk — APPROVED** [review]
I've worked on this area of the code in the past, so I might have caused the issue that this is fixing. It's an obvious fix, so not sure why it didn't occur to me last time I touched this area.

**Pull request merged**